db/db_impl.h

```diff
@@ -491,6 +491,8 @@ class DBImpl : public DB {
 
   uint64_t MinLogNumberToKeep();
 
+  uint64_t MinObsoleteSstNumberToKeep();
```

Comment:
The SST file whose number resides in `pending_outputs_` is not necessarily obsolete, is it?

Reply:
Right, the file with that number is not necessarily obsolete. I had trouble naming here. The situation is:

- non-obsolete SSTs are always kept
- obsolete SSTs are only kept if their number is greater than or equal to `MinObsoleteSstNumberToKeep`

Any alternative name suggestions would be appreciated :).

Reply:
Thanks @ajkr for the clarification. I think  `MinObsoleteSstNumberToKeep` is good.